### ISSUE TITLE
message menus for selecting a template

### ISSUE THREAD

**Issue by aoberoi** [nicolewhite/slack-meme#35]
now that slack supports [message menus](https://api.slack.com/docs/message-menus), i think you could use that to help the user compose a meme in a more intuitive way. i'm willing to contribute some code!

**Comment by aoberoi** [comment]
i'll wait to hear some feedback before working on this change 😄

**Comment by nicolewhite** [comment]
Yeah let's do it! Happy to give you direct merge access if you want to more or less take control of this project.

**Comment by aoberoi** [comment]
@nicolewhite awesome! i'd appreciate the collaborator offer very much! i don't want this to come off as a commitment to maintain or take over the project, but i'll help out where i can.

**Comment by aoberoi** [comment]
also relevant: slack now has [dialogs](https://api.slack.com/dialogs)

**Comment by nicolewhite** [comment]
Yeah I understand! I'll get back into this someday... 😭
